plugins/git/commands/git-summary.md

```diff
@@ -0,0 +1,12 @@
+---
+description: Show current branch, git status, and recent commits for quick context
+---
+
```

Comment:
can we adjust this to the ref implementaiton ## Sections which also include examples https://github.com/openshift-eng/ai-helpers/blob/main/plugins/hello-world/commands/echo.md